### ISSUE TITLE
Multiline comment breaks Prometheus format

### ISSUE THREAD

**Issue by schmengler** [run-as-root/magento2-prometheus-exporter#13]
If activating `magento_cronjob_broken_count', Prometheus cannot parse these lines:
```
# HELP magento_cronjob_broken_count_total Magento 2 CronJob Broken Count. 
Broken CronJobs occur when when status is pending but execution_time is set.
```

For multiline help texts, we should probably add `#` before each line

**Comment by schmengler** [comment]
Resolved by #11 (did not see that one)

**Comment by DavidLambauer** [comment]
Thanks for reporting this Fabian! Merged the PR.
